tests/federation/test_federation_catch_up.py

```diff
@@ -416,6 +418,9 @@ def wake_destination_track(destination):
         # ASSERT (_wake_destinations_needing_catchup):
         # - all remotes are woken up, save for zzzerver
         self.assertNotIn("zzzerver", woken)
-        self.assertEqual(set(woken.keys()), set(server_names[:-1]))
-        # check that all destinations were woken exactly once
-        self.assertEqual([value for value in woken.values() if value != 1], [])
+        # - all destinations are woken exactly once
+        # (assertCountEqual has a misleading name — it checks that the counts
+        #  of each item in the collections are not the same.
+        #  It does not merely check the lengths of the collections are equal,
+        #  as the name implies.)
```

Comment:
I don't think this comment is useful, `assertCountEqual` is a standard method from unittest.